macaroonbakery/identity.py

```diff
@@ -72,3 +72,48 @@ def allow(self, acls):
             if self._identity == acl:
                 return True
         return False
+
+
+class IdentityClient(object):
+    __metaclass__ = abc.ABCMeta
+
+    @abc.abstractmethod
+    def identity_from_context(self, ctx):
+        ''' Returns the identity based on information in the context.
```

Comment:
Please document the argument and return types.

Reply:
Return type still not formally documented.

Reply:
done.